app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  include WordpressEmbeddable
+
+  def test
+    render inline: "<% provide :title, 'Test' %><h1>Test Successful</h1>", layout: 'application'
```

Comment:
For inline layouts, it does, otherwise it renders the HTML with no layout at all. If I created an actual file in the views folder with the contents, it would be unnecessary, but this really only exists so that we have a simple page that will render within an embedded wordpress template without other dependencies (like all the legacy loan stuff being working).